ui/src/dashboards/components/DashboardPage.tsx

```diff
@@ -137,13 +141,6 @@ class DashboardPage extends Component<Props, State> {
   public componentDidUpdate(prevProps: Props) {
     const {autoRefresh} = this.props
 
-    const prevPath = getDeep(prevProps.location, 'pathname', null)
-    const thisPath = getDeep(this.props.location, 'pathname', null)
-
-    if (prevPath && thisPath && prevPath !== thisPath) {
-      this.getDashboard()
-    }
-
```

Comment:
I bet this was left over from when we had a quick dashboard switcher.

https://github.com/influxdata/influxdb/issues/13919

Time is a flat circle.

Reply:
omgosh you're right